cowprotocol/services
# Playground check

# Description
New CI job added to verify if Playground is working on current code.

# Changes
Added `playground-check.yaml` workflow file which starts Playground docker containers and runs dedicated test script: `test_playground.sh`. Run schedule is set to once a day or manually on chosen branch. Execution takes around 20min.

Test script tries to swap USDC for WETH tokens using Rest API. Additional calls using `cast` cli tool for wrapping ETH and submitting allowance. Signing is done also using `cast` cli tool in EIP712 scheme.

## How to test
It is enough to run `playground-check` job.
To test locally follow the instructions to start Playground in `playground/README.md` file, then run `playground/test_playground.sh` script.
